.eslintrc

```diff
@@ -65,6 +66,7 @@
     "no-use-before-define": [2, "nofunc"],
     "no-loop-func": [2],
     "no-console": [0],
-    "no-unused-labels": [2]
+    "no-unused-labels": [2],
+    "no-useless-escape": [0]
```

Comment:
how badly do we break this rule?

Reply:
this bad:

![image](https://user-images.githubusercontent.com/6675409/35709769-11551bf6-0782-11e8-872f-4bfdb57ebb33.png)

Reply:
... and they are not auto-fixable (using `eslint . --fix`) unfortunately.

Reply:
... and I don't want to waste time with linting stuff. I really think we should be switching to `standardjs` and be done with it (cc https://github.com/plotly/plotly.js/issues/950) :stuck_out_tongue: